src/.nuget/Microsoft.NETCore.ILDAsm/runtime.OSX.Microsoft.NETCore.ILDAsm.props

```diff
@@ -19,11 +12,10 @@
     <AdditionalLibPackageExcludes Include="%2A%2A\%2A.dwarf" />
     <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dylib" />
     <AdditionalSymbolPackageExcludes Include="%2A%2A\%2A.dll" />
-    <ArchitectureSpecificNativeSymbol Include="..\..\_.pdb" />
+    <ArchitectureSpecificNativeSymbol Include="..\_.pdb" />
```

Comment:
Nit: we should have a property for this sort of thing.  I think we did in corefx.  That can be addressed separately.  /cc @dagood